src/frontend/src/App.tsx

```diff
@@ -1,15 +1,49 @@
-import React from "react";
-import snakeoil from "./images/snakeoil.jpg";
-import "./App.scss";
-
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={snakeoil} className="App-logo" alt="logo" />
-      </header>
-    </div>
-  );
-};
+import React from 'react';
+import styled from 'styled-components';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+
+import Navigation from './navigation/navigation';
+import UserManagement from './user-management/user-management';
+import Home from './home/home';
+
+function App() {
+    return (
+        <Router>
+            <Container>
+                <Route>{({ location }) => <Navigation path={location.pathname} />}</Route>
+                <Switch>
+                    <MainContent>
+                        <Route path='/users'>
+                            <UserManagement />
+                        </Route>
+                        <Route path='/reservation'>
+                            <Home />
+                        </Route>
+                        <Route path='/procedures'>
+                            <p>procedures</p>
+                        </Route>
+                        <Route path='/history'>
+                            <p>history</p>
```

Comment:
asi mam len vyhrady k tomu, ze pisme to rovno po slovensky nech sa to potom nemusi prepisovat uz

Reply:
Ok, som si len dal narychlo nieco nech mozem otestovat ten react router.